### PR TITLE
[108] Added sync with Google calendar

### PR DESCRIPTION
**Pull request by ozhurova**
Added sync with Google calendar
image
image

### CODE REVIEW

File: src/app/pages/calendar-page/calendar-page.component.ts [hunk 1]
```diff
@@ -30,31 +32,52 @@ export class CalendarPageComponent implements OnInit, OnDestroy {
...
+  //     this.end = this.calendar.view.activeEnd.toISOString();
+  //     // if (this.userService.user._roles.includes('Admin')){
+  //     // this.calendarEventService.getEventsByType(this.start, this.end, 1);
+  //     this.calendarEventService.eventSubject.subscribe((res: EventInput[]) => {
+  //       this.calendarOptions.events = res;
+  //     });
+  //   }
+
+  // }
+
```
Comment (mhasilau) [inline src/app/pages/calendar-page/calendar-page.component.ts:75]: pls del comments

File: src/assets/locales/en.json [hunk 2]
```diff
@@ -172,6 +172,8 @@
     "subText": "submit",
```
Comment (mhasilau) [inline src/assets/locales/en.json:172]: pls change the firsts letters to upper case on this file and ru.json

File: src/app/pages/calendar-page/calendar-page.component.ts (hunk 1 above)
Comment (ozhurova) [reply src/app/pages/calendar-page/calendar-page.component.ts]: done

File: src/assets/locales/en.json (hunk 2 above)
Comment (ozhurova) [reply src/assets/locales/en.json:172]: I cannot change all the buttons and therefore I will not change any

Comment (mhasilau) [reply src/assets/locales/en.json:172]: even the ones you created?

Comment (ozhurova) [reply src/assets/locales/en.json:172]: done all buttons with firsts letters to uppercase

**Review by mhasilau — APPROVED** [review]

**Review by Nicolai-boiko — APPROVED** [review]

**Review by vstrabko — APPROVED** [review]

**Review by PaulRomanov — APPROVED** [review]

**Pull request merged**